src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1762,31 +1768,31 @@ int JasmineGraphInstanceService::run(string profile, string masterHost, string h
     int connectionCounter = 0;
     pthread_mutex_init(&file_lock, NULL);
     pthread_t threadA[MAX_CONNECTION_COUNT];
-
+    struct instanceservicesessionargs instanceservicesessionargs1;
+    std::map<std::string, JasmineGraphHashMapLocalStore> graphDBMapLocalStores;
+    std::map<std::string, JasmineGraphHashMapCentralStore> graphDBMapCentralStores;
+    std::map<std::string, JasmineGraphHashMapDuplicateCentralStore> graphDBMapDuplicateCentralStores;
+    std::map<std::string, JasmineGraphIncrementalLocalStore> incrementalLocalStore;
+
+    instanceservicesessionargs1.graphDBMapLocalStores = graphDBMapLocalStores;
```

Comment:
Why do we need `1` at the end of the variable name `instanceservicesessionargs1`. We anyway create only one variable of this type.

Reply:
I think we might have used 1 to prevent name coalitions with the name of the struct itself, Changed it as `serviceArguments` in this PR https://github.com/miyurud/jasminegraph/pull/132/commits/d8d887f5453408266df16065d675be6691184b72

Reply:
Ok